crates/biome_js_analyze/tests/specs/nursery/useTrimStartEnd/valid.js

```diff
@@ -0,0 +1,25 @@
+const foo = bar.trimStart()
+const foo = bar.trimEnd()
+bar.trimStart?.()
+foo.trimStart()
+foo.trimStart?.()
+foo.trimEnd()
+// Not `CallExpression`
+new foo.trimLeft()
+// Not `MemberExpression`
+trimLeft()
+// `callee.property` is not a `Identifier`
+foo[\'trimLeft\']()
```

Comment:
This is still not valid syntax. You can just copy paste it in the console to see the problem.